.changelog/dependabot-changelog.sh

```
@@ -0,0 +1,220 @@
+    printf '          Title: "%s"\n' "$title"
+fi
+
+# Dependabot branch names look like this: "dependabot/github_actions/bufbuild/buf-setup-action-1.34.0"
+# The "github_actions" part can also be "go_modules" (and probably other things too).
+# For the filename, we'll omit the "dependabot/<lib type>/" part and use just what's left.
+branch_fn="$( sed -E 's|^[^/]+/[^/]+/||; s|/|-|g;' <<< "$head_branch" )"
+[[ -n "$verbose" ]] && printf 'Branch Filename: "%s"\n' "$branch_fn"
+
+# This script requires another script that must be in the same directory.
+# To consistently find them, we'll need to know the absolute path to the dir with this script.
+where_i_am="$( cd "$( dirname "${BASH_SOURCE:-$0}" )"; pwd -P )"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)